app/controllers/admin/words_controller.rb

```diff
@@ -0,0 +1,58 @@
+class Admin::WordsController < ApplicationController
+  before_action :find_category, only: [:index, :show, :create, :edit, :update, :destroy]
+  def find_category
+    @category = Category.find_by(params[:category_id])
+  end
+  def index
+    # @words = @category.words.paginate(page: params[:page], per_page: 10)
+      @words = Word.all
+  end
+
+  def show
+    #shows one id
+    @word = Word.find(params[:id])
+end
+
+  def new
+    @word = Word.new
+  end
+
+  def create
+    @word = @category.words.build(word_params)
+
+    if @word.save!
+      flash[:success] = "Word created!"
+      redirect_to admin_category_word_url(@category, @word)
```

Comment:
Change redirect_to URL from ADMIN WORD SHOW page to ADMIN WORD INDEX, so that we avoid errors.
If we force your controller to go to SHOW PAGE, you can't display many words inside because SHOW only allows one (1) item at a time.

We need this to be in INDEX PAGE for ADMIN WORD because you can display many words inside of one (1) category,